cmd/enr.go

```diff
@@ -64,20 +64,18 @@ func runNewENR(w io.Writer, config p2p.Config, dataDir string, verbose bool) err
 		return err
 	}
 
-	localEnode, db, err := p2p.NewLocalEnode(config, key)
+	enrStr, err := createENR(key, config)
 	if err != nil {
-		return errors.Wrap(err, "failed to open peer DB")
+		return err
 	}
-	defer db.Close()
 
-	newEnr := localEnode.Node().String()
-	_, _ = fmt.Fprintln(w, newEnr)
+	_, _ = fmt.Fprintln(w, enrStr)
 
 	if !verbose {
 		return nil
 	}
 
-	r, err := p2p.DecodeENR(newEnr)
+	r, err := p2p.DecodeENR(enrStr)
```

Comment:
another round of encoding and decoding, feel like `createENR` should return `enr.Record`

Reply:
sure